examples/web_agent/README.md

```diff
@@ -0,0 +1,47 @@
+# WEBAGENT
+
+This is an agent that uses python to connect to Airtop, create sessions, execute prompts on the site and store the data in a local database.
+
+## PREREQUISITES
+
+- PYTHON 3.10
+
+To install, just run the following after downloanding the code
+
+`poetry install`
```

Comment:
@nrotstan @kyleparrott @kvindascr @pciporin25, should `poetry` be our recommendation on recipes? Just making sure we are establishing the right pattern here for the projects.

Reply:
I don't know that there is one "right" way to manage python dependencies today and there doesn't seem to be much consensus across some of the larger projects out there.  

Poetry seems like a good choice to me, unless we find a compelling reason to change in the future. Which ever tool we choose to go with we should be consistent with across all of our recipes.